pkg/agent/cniserver/ipam/antrea_ipam_controller.go

```diff
@@ -63,9 +63,9 @@ func podIndexFunc(obj interface{}) ([]string, error) {
 		return nil, fmt.Errorf("obj is not IPPool: %+v", obj)
 	}
 	podNames := sets.NewString()
-	for _, IPAddress := range ipPool.Status.IPAddresses {
-		if IPAddress.Owner.Pod != nil {
-			podNames.Insert(k8s.NamespacedName(IPAddress.Owner.Pod.Namespace, IPAddress.Owner.Pod.Name))
+	for _, ipAddr := range ipPool.Status.IPAddresses {
```

Comment:
Why use `Addr` here? Existing `IPAddress` or `ipAddress` should be OK.

Reply:
According to our code convention, local variables should start with lower case, so I made the change. But if you prefer `ipAddress` over `ipAddr`, I can change to `ipAddress`.